cpu/esp32/Makefile.include

```diff
@@ -1,3 +1,6 @@
+# thin archives trigger a reboot loop - see #12258, #12035
```

Comment:
Maybe it's a bit nitpickin, but it shouldn't be the first line before all other general default settings. From my point of view, a good place would be after `CFLAGS` and before `LINKFLAGS`

Reply:
My reasoning to have it on top was so that it is apparent this is just a quick and hopefully temporary workaround.
But I can move it further down.

Reply:
> My reasoning to have it on top was so that it is apparent this is just a quick and hopefully temporary workaround.

That was what I thought. But if we can't find the reason for the problem, it will become persistent and will not be touched again.